spec/services/location_suggestion_spec.rb

```diff
@@ -61,6 +65,38 @@ def stub_query(status: 200, predictions: {}, error_message: nil)
       end
     end
 
+    context 'caching' do
+      before do
+        query_stub
+      end
+
+      it 'caches suggestions for a period of time' do
+        allow(Rails.cache).to receive(:read).and_call_original
+        allow(Rails.cache).to receive(:write).and_call_original
+
+        result = suggest!
+        expect(result.count).to eq 3
+        expect(Rails.cache).to have_received(:read).once
```

Comment:
Do we need to assert on these calls? Whether the request gets made or not feels like the salient thing